ml-data-hub-plugin/src/test/resources/update-indexes/final-database.xml

```diff
@@ -129,5 +129,12 @@
       <range-value-positions>true</range-value-positions>
       <invalid-values>reject</invalid-values>
     </range-path-index>
+    <range-path-index>
```

Comment:
It looks like these duplicate files are only needed for UpdateIndexesTaskTest. But after looking at that test, I don't think it's testing anything specific to DHF - it's just re-testing ml-app-deployer functionality.  DHF can safely assume that if it defines indexes in a database file, then ml-app-deployer will deploy them correctly.

I think we should get rid of these duplicate files. My first choice is to delete them and delete UpdateIndexesTaskTest. My second choice is to delete them and modify UpdateIndexesTaskTest to copy the real files from ../marklogic-data-hub/src/main/resources/

Reply:
UpdateIndexesTaskTest ensures that the payload generated by 'hubSaveIndexes' task is correct and can be deployed without issues. Also the files (final-database.xml and staging-database.xml) are not duplicates of each other, there are additional indexes in them

Reply:
Okay, sounds good.